index.html

```diff
@@ -1,108 +1,159 @@
+            </div>
+
+            {{# if facultyData}}
+              <div class="col-sm-5">
+                <h2> Faculty </h2>
+                <canvas id="faculty" height="400"></canvas>
+              </div>
+            {{/if}}
+
+            <div class="col-sm-12">
+              <h2> Helpers </h2>
+              <canvas id="helpers" height="400"></canvas>
+            </div>
+
+            <div class="col-sm-12">
+              <h2> Participants <h2>
+              <input id="participantsFilter" type="text" placeholder="Search.."></input>
+              <div id="participants"></div>
+            </div>
+          </div>
+
+          <div id="partners" class="col-sm-3">
+                <h2>Partners</h2>
+                {{#each partners}}
+                <a href={{this.url}}> <img class={{this.type}} src={{this.src}} /> </a>
+                {{/each}}
+                <iframe src="http://www.facebook.com/plugins/likebox.php?href=https%3A%2F%2Fwww.facebook.com%2Frosedu.org&amp;width=200&amp;height=290&amp;colorscheme=light&amp;show_faces=true&amp;header=true&amp;stream=false&amp;show_border=false" scrolling="no" frameborder="0" style="border:none; overflow:hidden; width:200px; height:290px;" allowTransparency="true"></iframe>
+              </div>
+
+        {{else}}
+            <div class="description col-sm-9">
+              <strong>ROSEdu</strong> pregătește cea de a 10-a ediție a
+              <a href="https://www.facebook.com/events/569614456477589/"><strong>Linux Install Fest</strong></a>.
+              <div>
+                Ai posibilitatea, alături de noi, să îți instalezi orice distribuție Linux pe laptopul sau computerul tău.
+                Vom fi alături de tine pentru a rezolva problemele legate de drivere, partiționare sau migrarea datelor între sisteme de operare diferite.
+              </div>
+              <div>
+                <strong>Linux Install Fest</strong> își propune să aducă împreună timp de câteva ore oameni pasionați din diverse comunități Linux.
+              </div>
+              <div class="note">
+                *Dacă ai un sistem desktop îți poți aduce doar unitatea. Noi vom pune la dispoziție monitoare, tastaturi și mouse-uri.
+              </div>
+              <div>
+                <strong>Când</strong>
+              </div>
+              <div class="time">
+                Sâmbătă, <strong> 10 Octombrie </strong> - Între <strong> 10:00 și 16:00 </strong>.
+              </div>
+              <div class="date">
+              </div>
+              <div>
+                <strong>Unde</strong>
+                <div>
+                  <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2848.7075686723388!2d26.0522087544486!3d44.43946460791785!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40b201c1c2998f79%3A0x1acb42f08800db7a!2sBiblioteca+Central%C4%83!5e0!3m2!1sen!2sro!4v1444164367417" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
```

Comment:
Din nou, tot pe alt isssue, asta se mai schimba din rectorat in hol EC, deci ar putea fi variabila si ea